package.json

```diff
@@ -37,6 +37,7 @@
   },
   "dependencies": {
     "chalk": "^2.4.1",
+    "esm": "^3.2.25",
```

Comment:
so we want to avoid shipping loaders/interpreters/compilers with the module. in the documentation for the other compatible options, we say that a user must have that installed themselves before specifying one for use. we'll want to follow that model.